src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+

```

Comment:
in mate we don't leave empty lines after class definiton

Reply:
not fixed